### ISSUE TITLE
Switch focus on next window before spawn new one?

### ISSUE THREAD

**Issue by tmpm697** [ifreund/river#325]
I logged the issue here, but don't know if it related to `river` or `kile`: https://gitlab.com/snakedye/kile/-/issues/2

steps:

1. start river with `kile` (kile-wl) as default-layout
2. spawn two alacritty windows, one has ls -l command, one has none
3. spawn firefox window (I start from combined keys mapped with riverctl in river's init)
4. have test.sh with this line: swaylock --color 00000000 --no-unlock-indicator
5. from one of terminal, issue this command: ./test.sh to run swaylock
6. observe screen now have firefox appears and took top stack of screen (I use full mode of kile, so look like window now focus on firefox and wait for user password to unlock.
7. but when I enter password, window focus switches back to original terminal that issued ./test.sh

Basically I understand that when I do swaylock, it will spawn new window to display lock and block input ..etc.
When I invoke test.sh to call swaylock, I want focus keep staying in current window and not change the focus to another window.

**Comment by tmpm697** [comment]
pls check video: https://imgur.com/gallery/zrlYhpU

I have three windows: 2 alacritty and one firefox.

cat test.sh
```
sleep 1
swaylock --color 00000000 --no-unlock-indicator
sleep 1
```

**Comment by ifreund** [comment]
Since this can't be reproduced with rivertile, I believe this is an issue with kile.
